.github/workflows/lint-tests.yml

```diff
@@ -0,0 +1,162 @@
+name: Coding Standards and Tests
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+
+jobs:
+  lint:
+    name: "Coding Standards"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run coding standards check
+        run: npm run lint
+
+  test-e2e:
+    needs: [lint]
+    name: "E2E tests (PHP 7.4, WordPress latest, with code coverage)"
+    runs-on: ubuntu-latest
+
+    env:
+      NODE_ENV: e2e
+      WP_VERSION: 5.7
```

Comment:
Why is this hardcoded to 5.7? Travis just uses latest 
https://github.com/xwp/wp-foo-bar/blob/develop/.travis.yml#L71